web/src/components/organisms/Asset/AssetList/index.tsx

```diff
@@ -0,0 +1,206 @@
+import { useParams } from "react-router-dom";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import CustomTag from "@reearth-cms/components/atoms/CustomTag";
+import DownloadButton from "@reearth-cms/components/atoms/DownloadButton";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import {
+  ListToolBarProps,
+  ProColumns,
+  OptionConfig,
+  TableRowSelection,
+  TablePaginationConfig,
+} from "@reearth-cms/components/atoms/ProTable";
+import { UploadProps } from "@reearth-cms/components/atoms/Upload";
```

Comment:
Sorry if I have been allowing it up until now in previous PRs, but actually organisms should be made up of ONLY molecules.
Please clean up this and any other organism in this PR that imports atoms

Reply:
Done!